.github/workflows/release-drafter.yml

```diff
@@ -0,0 +1,32 @@
+name: Release Drafter
+
+on:
+  push:
+    branches:
+      - main
+
+  # pull_request event is required only for autolabeler
+  pull_request:
+    # Only following types are handled by the action, but one can default to all as well
+    types: [opened, reopened, synchronize]
+  # pull_request_target event is required for autolabeler to support PRs from forks
+  # pull_request_target:
+  #   types: [opened, reopened, synchronize]
+
+
+jobs:
+  permissions:
```

Comment:
@d10n Did you mean to put this at the top level? I think moving it would solve the two token permission related failures.

Reply:
Thanks - this line was accidentally included - the job-specific permissions are applied below